docs/tools/vdb_table/data/milvus.json

```diff
@@ -133,21 +133,21 @@
     "comment": ""
   },
   "sharding": {
-    "support": "",
-    "source_url": "",
+    "support": "full",
+    "source_url": "https://milvus.io/docs/glossary.md#Sharding",
     "comment": ""
   },
   "doc_size": {
-    "bytes": 0,
-    "unlimited": false,
+    "bytes": "Unlimited",
+    "unlimited": true,
     "source_url": "",
```

Comment:
any link to docs that explains what "ultimited" means here? for example, how doc size translates to ram/disk consumption?

Reply:
I added in comments. Since Milvus is distributed arch, you can add ram/disk resource as much as you can to archieve unlimited data.

Reply:
Understood - still, pointing the reader to some related documentation is helpful, perhaps:
https://milvus.io/docs/product_faq.md#What-is-the-maximum-dataset-size-Milvus-can-handle

Reply:
Yes you are correct